test/hotspot/jtreg/compiler/lib/ir_framework/Setup.java

```diff
@@ -28,9 +28,11 @@
 
 /**
  * This annotation is used to identify Setup methods. These can be used to compute arbitrary arguments for a test
- * method (see {@link Test}), as well as set fields. A test method can use a Setup method, by specifying it in a
- * {@link Arguments} annotation. A setup method can optionally take a {@link SetupInfo} as an argument. The for the
- * arguments for the test methods are returned in an Object[].
+ * method (see {@link Test}), as well as to set field values. A test method can use a setup method, by specifying it in a
+ * {@link Arguments} annotation. A setup method can optionally take a {@link SetupInfo} as an argument. The
```

Comment:
Wrapping was off after the suggestions:
```suggestion
 * method (see {@link Test}), as well as to set field values. A test method can use a setup method, by specifying
 * it in a {@link Arguments} annotation. A setup method can optionally take a {@link SetupInfo} as an argument. The
```